sentry-core/src/main/java/io/sentry/SentryEvent.java

```diff
@@ -19,6 +26,16 @@
   private String logger;
   private SentryValues<SentryThread> threads;
   private SentryValues<SentryException> exceptions;
+  private SentryLevel level;
+  private String transaction;
+  private String environment;
+  private User user;
+  private Request request;
+  private SdkVersion sdkVersion;
+  private List<String> fingerprint = new ArrayList<>();
+  private List<Breadcrumb> breadcrumbs = new CopyOnWriteArrayList<>();
```

Comment:
Event doesn't need `CopyOnWrite`. It is not thread-safe so we can use simple(cheaper) types here.